lib/static/new-ui/features/suites/components/SuitesPage/index.module.css

```diff
@@ -46,6 +46,6 @@
     background-color: white;
     position: sticky;
     top: 0;
-    z-index: 1;
+    z-index: 10;
```

Comment:
hm?

Reply:
DiffViewer has absolutely positioned elements, so z-index 1 is no longer enough for the sticky-header.